pkg/scale/encode_test.go

```diff
@@ -4,12 +4,48 @@
 package scale
 
 import (
+	"bytes"
 	"math/big"
 	"reflect"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Let me know if you feel this is inappropriate to use to keep zero-dependency on scale.
Although right now since it's part of Gossamer it doesn't change anything really.

Reply:
Tagging @timwu20 here, if we are planning to migrate scale out of gossamer might be better to keep dependencies out?

Reply:
https://github.com/ChainSafe/gossamer/pull/2683#discussion_r942741296